storio-common-annotations-processor/src/test/java/com/pushtorefresh/storio/common/annotations/processor/ProcessingExceptionTest.java

```diff
@@ -0,0 +1,41 @@
+package com.pushtorefresh.storio.common.annotations.processor;
+
+
+import static org.junit.Assert.assertEquals;
+import static org.powermock.api.mockito.PowerMockito.mock;
+import static org.powermock.reflect.Whitebox.getInternalState;
+
+import javax.lang.model.element.Element;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class ProcessingExceptionTest {
+	
+	private Element elementMock;
+	
+	@Before
+	public void setUp() throws Exception {
+		elementMock = mock(Element.class);

```

Comment:
Hm, looks like you have indentation `== 8 spaces`, can you please change it to `4`?